### DISCUSSION TITLE
Kinematic Hardening Material Model

### DISCUSSION THREAD

**Discussion by ppandit95** [peridigm/peridigm#193] [General]
Dear Peridigm Community

I wish to simulate tensile test with kinematic hardening correspondence model which I believe can be developed from "Isotropic Hardening Correspondence" model but for that I need accumulated stress which needs to be algebraically manipulated.So any headers in this regard will be helpful in development.

Many Thanks
Pushkar

**Comment by djlittl** [comment]
The tensile_test example computes engineering stress and strain by mimicking a laboratory experiment.  The forces (reaction forces) are tracked at the ends of the bar, and a compute class is used to track two points on the bar in the same way that a strain gauge would work in the lab.  The force and displacement data is converted to engineering stress and strain with a simple post-processing operation.  Basically divide the force by the cross-sectional areas and compute engineering strain using the initial and deformed length of the "strain gauge".

**Reply by ppandit95** [reply]
Thanks @djlittl for clarifying about the tensile test example and as I understand the "Kinematic Hardening Correspondence" model can be developed from "Isotropic Hardening Correspondence" model
